package.json

```diff
@@ -9,6 +9,7 @@
     "react-bootstrap": "^0.32.4",
     "react-content-loader": "^4.2.1",
     "react-dom": "^16.8.0",
+    "react-router-dom": "^5.0.1",
```

Comment:
We shouldn't need `react-router-dom` for this. Let's uninstall this back out.

Reply:
Put it in do be able to use the styles written for that in the mdx file, but I can probably do it differently.